tests/test_roles.py

```diff
@@ -0,0 +1,18 @@
+from http import HTTPStatus
+
+
+def test_list(client):
+    assert client.get("/roles/").status_code == HTTPStatus.OK
+
+
+def test_create(client):
+    assert client.get("/roles/create").status_code == HTTPStatus.OK
+
+
+def test_edit(client):
+    assert client.get("/roles/edit/1").status_code == HTTPStatus.OK
+
+
+def test_delete(client):
+    response = client.post("/roles/delete", data={"id": 1})
+    assert response.headers["Location"] == "http://localhost/roles/"
```

Comment:
@krzyk I liked the way it was done. You are using the HTTPStatus constants of python instead of code (Example: HTTPStatus.OK instead of 200).
Anyway, please insert a new line at the end of the file to get more organized.